### PR TITLE
Release GitHub action

### PR DESCRIPTION
**Pull request by dennys-bd**
image

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -11,25 +11,23 @@
...
+    AWS_BUCKET_NAME = config("AWS_BUCKET_NAME")
+    AWS_ACCESS_KEY_ID = config("AWS_ACCESS_KEY_ID")
+    AWS_SECRET_ACCESS_KEY = config("AWS_SECRET_ACCESS_KEY")
+    c.run(
+        "s3cmd sync {}/ s3://{} --acl-public --delete-removed "
+        "--guess-mime-type --access_key={} --secret_key={}".format(
+            FOLDER,
+            AWS_BUCKET_NAME,
+            AWS_ACCESS_KEY_ID,
+            AWS_SECRET_ACCESS_KEY,
```
Comment (dennys-bd) [inline fabfile.py:23]: Had problems using fab inside a venv inside gha, it was not identifying fab so I chose to remove the venv as default.

Comment (fjsj) [reply fabfile.py:23]: The problem is this may end up being printed in gha, no? Please double-check.

**Review by fjsj — CHANGES_REQUESTED** [review]
LGTM, but there's a security concern. Please check comment.

File: fabfile.py (hunk 1 above)
Comment (dennys-bd) [reply fabfile.py:23]: GHA doesn't print secrets
image

**Review by fjsj — APPROVED** [review]

**Pull request merged**